packages/taquito-rpc/src/taquito-rpc.ts

```diff
@@ -801,9 +801,13 @@ export class RpcClient implements RpcClientInterface {
    * @param options contains generic configuration for rpc calls
    *
    * @description Computes the serialized version of a data expression using the same algorithm as script instruction PACK
+   * Note: Users should be cautious if signing data packed with the RPC. They should always verify the packed bytes before signing them. 
+   * This precaution is to avoid signing unwanted operations in cases where the RPC could be malicious or compromised. 
+   * The safest solution to pack and sign data would be to use the `packDataBytes` function available in the `@taquito/michel-codec` package.
```

Comment:
Use same rewrite as I provided above please.

Reply:
Thank you, I made the change